orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/strategy/Strategy.java

```diff
@@ -62,6 +63,8 @@ public boolean replacesBasicSteps() {
         return builder.composeHighlanderFlow(stage);
       case ROLLING_PUSH:
         return builder.composeRollingPushFlow(stage);
+      case RECREATE_UPDATE:
+        return builder.composeRollingPushFlow(stage);
```

Comment:
This looks like a typo

Reply:
It doesn't have major differences between OnDeleteUpdate and RolllingRedBlack strategies.  It is just resize happen before or after the update the controller definition.  OnDeleteUpdate also require to provide `kind`.  Do you want me to re-use RollingRedBlack strategy?